opal/runtime/opal_progress.h

```diff
@@ -195,6 +195,9 @@ static inline bool opal_progress_spin(volatile bool *complete)
     return false;
 }
 
+/* shutdown the async progress thread. Do nothing if disabled */
```

Comment:
This function should not be visible outside `opal_progress.c`. Please make it static.

Reply:
This function will be called early in the finalize of `ompi_mpi_finalize()`, which is a solution emerged from #13073 to solve the race during tearing down between the progress thread and the `mca_btl_sm_component` destructor executed by the main thread.

Reply:
My bad, I misread the function and I thought it was the thread progress function.